scripts/mixins/thaumcraft_dev.zs

```diff
@@ -0,0 +1,20 @@
+#loader mixin
+
+import native.java.util.ArrayList;
+import native.net.minecraft.item.ItemStack;
+import mixin.CallbackInfoReturnable;
+
+#mixin Mixin
+#{targets: "thaumcraft.common.lib.crafting.ThaumcraftCraftingManager"}
+zenClass MixinThaumcraftCraftingManager {
+    #mixin Static
+    #mixin Inject
+    #{
+    #    method: "generateTagsFromRecipes",
+    #    at: {value: "HEAD"},
+    #    cancellable: true
+    #}
+    function disableOredictMeltingCasting(stack as ItemStack, history as ArrayList, cir as CallbackInfoReturnable) as void {
```

Comment:
Probably typo in function name, should be something like `skipGenerateAspectsFromRecipes`